app/themes/federal.tsx

```diff
@@ -165,18 +165,18 @@ theme.typography = merge(theme.typography, {
     fontWeight: "bold",
   }),
   body1: createTypographyVariant(theme, {
-    fontSize: [14, 16],
-    lineHeight: [22, 24],
+    fontSize: 14,
+    lineHeight: [24],
     fontWeight: "regular",
   }),
```

Comment:
I think it should be fontSize: [16] here, no?

Reply:
I am keeping this for another PR, I need to see how to see this in the sketch file.

Reply:
I extracted the commit